packages/aws-cdk-lib/aws-events-targets/README.md

```diff
@@ -367,6 +367,51 @@ const rule = new events.Rule(this, 'OtherRule', {
 });
 ```
 
+## Invoke an AppSync GraphQL API
+
+Use the `AppSync` target to trigger an AppSync GraphQL API. You need to
+create an `AppSync.GraphqlApi` configured with `AWS_IAM` authorization mode.
+
+The code snippet below creates an AppSync GraphQL API target that is invoked every hour, calling the `publish` mutation.
+
+```ts
+import * as appsync from 'aws-cdk-lib/aws-appsync';
+declare const api: appsync.GraphqlApi;
```

Comment:
I think we should include the instantiation of `GraphqlApi` to make the example more complete, especially if there are properties that need to be configured to a certain value, e.g. `AWS_IAM` for authorization